### DISCUSSION TITLE
关于强制https 的问题

### DISCUSSION THREAD

**Discussion by cn-jiangfei** [AlistGo/alist#5167] [Ideas]
现在如果开启了HTTPS不强制的话， http 和 https 可以并存，同时使用，但是开启强制HTTPS 以后，http 就无法访问了，为何不是跳转 https 呢？既然强制就是需要跳转，如果不想使用http 则  http_port 可以设置为 -1 这样 http 就不可以访问了。 希望这里可以改进一下。或者强制https 以后还需要其他设置才能实现 http 跳转 https ?

**Comment by anwen-anyi** [comment]
强制从http跳转https这得用`Nginx`来控制吧？AList程序估计不能

**Reply by cn-jiangfei** [reply]
那就是说，只能通过反向代理来实现了！

**Comment by xhofe** [comment]
已在 6f6d20e1ba32c50c473c5751941edb1581f88aad 中修复。
